package.json

```
@@ -12,8 +12,8 @@
     "tsc:noemit": "tsc --noemit",
     "lint": "tslint -p . -c tslint.json -t verbose",
     "lint-autofix": "tslint -p . -c tslint.json -t verbose --fix",
+    "generate:pagopa-api-definitions": "rimraf definitions/pagopa && mkdir -p definitions/pagopa && gen-api-models --api-spec https://wisp2.pagopa.gov.it/pp-restapi-CD/v2/api-docs --out-dir ./definitions/pagopa --no-strict --request-types --response-decoders --default-error-type string",
```

`string` is not a type


```suggestion
    "generate:pagopa-api-definitions": "rimraf definitions/pagopa && mkdir -p definitions/pagopa && gen-api-models --api-spec https://wisp2.pagopa.gov.it/pp-restapi-CD/v2/api-docs --out-dir ./definitions/pagopa --no-strict --request-types --response-decoders --default-error-type String",
```

---

[UPDATE] io-utils needs a fix about using default types for error responses (when not specified) https://github.com/teamdigitale/io-utils/issues/101